includes/application/forwardApplication.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/*
+ *
+ *  to forward application if not processed in certain days.
+ */
+
+session_start();
+
+if ( !( isset( $_SESSION['Username'] ) && isset( $_SESSION['Name'] ) ) )
```

Comment:
__[code_sniffer]__
* Expected 0 spaces after opening bracket; 1 found
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 1

<!-- Comment by [SideCI](https://sideci.com) -->